geode-core/src/test/java/org/apache/geode/internal/cache/PartitionedRegionTest.java

```diff
@@ -221,22 +220,6 @@ public void clearShouldNotThrowUnsupportedOperationException() {
     spyPartitionedRegion.clear();
   }
 
-  @Test(expected = CacheClosedException.class)
-  public void clearShouldThrowCacheClosedExceptionIfShutdownAll() {
```

Comment:
Is the deleted test case covered else where? Or it no longer valid?

Reply:
This is based on the earlier clear messaging. Thats not valid anymore.